src/api/data_types/chunking/mobile_app.rs

```diff
@@ -20,5 +20,5 @@ pub struct AssembleMobileAppResponse {
     pub state: ChunkedFileState,
     pub missing_chunks: Vec<Digest>,
     pub detail: Option<String>,
-    pub artifact_id: Option<String>,
+    pub artifact_url: Option<String>,
```

Comment:
Ah wait, this produces a behavior change. We now are trying to deserialize a field named `artifact_url` instead of `artifact_id`